core/src/Alert.html

```diff
@@ -39,7 +39,7 @@
       const processedData = this.processTextAndLinks(text, links);
       this.set({
         settings: Object.assign(this.get().settings, {
-          text: processedData.sanitizedText
+          alertText: processedData.sanitizedText
```

Comment:
What happens if someone initializes the Alert with "settings.alertText" filled with (malicious) text?

Reply:
fixed